packages/core/test_resources/pubspec_strings.yaml

```diff
@@ -0,0 +1,11 @@
+name: test
+
+flutter_gen:
+  output: lib/gen/ # Optional (default: lib/gen/)
+  line_length: 80   # Optional (default: 80)
+
+  strings:
+    inputs:
+      - assets/strings/ka.json
+      - assets/strings/en.json
+      - assets/strings/en.json
```

Comment:
What will happen if you have duplicate input keys but they have different values? In your test case, all keys and values are the same.